includes/wf_crm_webform_preprocess.inc

```diff
@@ -549,7 +549,8 @@ class wf_crm_webform_preprocess extends wf_crm_webform_base {
         $taxSettings = CRM_Core_BAO_Setting::getItem(CRM_Core_BAO_Setting::CONTRIBUTE_PREFERENCES_NAME, 'contribution_invoice_settings');
 
         if ($taxSettings['tax_display_settings'] != 'Do_not_show') {
-          $item['label'] .= ' (' . t('includes !rate @tax', array('!rate' => $this->tax_rate . '%', '@tax' => $taxSettings['tax_term'])) . ')';
+          $tax_rate = number_format($this->tax_rate, 2); //strip trailing zeros from tax_rate e.g 20.00000% => 20.00%
```

Comment:
Please add the comment in a separate line following instructions in https://www.drupal.org/node/1354